gulpfile.js

```diff
@@ -0,0 +1,232 @@
+// This file is in charge of the build process:
```

Comment:
Already discussed with Michaël.
I don't plan to use Grunt... But if someone (external to the team) is ready to migrate everything I did with Gulp to Grunt, why not?